src/linkage.md

```diff
@@ -301,8 +298,9 @@ r[link.unwinding.prohibited]
 If a Rust artifact is potentially unwinding, then all its crates must be built with the `unwind` [panic strategy].
```

Comment:
```suggestion
If a Rust artifact is potentially unwinding, then all its crates must be built with the `unwind` [panic strategy].
Otherwise, unwinding can cause undefined behavior.
```